resources/qml/TertiaryButton.qml

```diff
@@ -9,13 +9,11 @@ import Cura 1.1 as Cura
 
 Cura.ActionButton
 {
-    shadowEnabled: true
-    shadowColor: enabled ? UM.Theme.getColor("secondary_button_shadow"): UM.Theme.getColor("action_button_disabled_shadow")
     color: "transparent"
     textColor: UM.Theme.getColor("text_link")
     outlineColor: "transparent"
-    disabledColor: UM.Theme.getColor("action_button_disabled")
+    outlineDisabledColor: "transparent"
+    disabledColor: "transparent"
     textDisabledColor: UM.Theme.getColor("action_button_disabled_text")
-    hoverColor: "transparent"
-    underlineTextOnHover: true
-}
+    hoverColor: UM.Theme.getColor("secondary_button_hover")
```

Comment:
Tertiary buttons should not use secondary button theme elements. It would be better to duplicate the theme entry if they happen to be the same in the current design.

Reply:
Agreed, resolved in: https://github.com/Ultimaker/Cura/pull/10071/commits/acf3f2d02a1a0cf393cf6e10d381a4a28dd1376e